website/content/en/docs/getting-started/_index.md

```diff
@@ -1,21 +1,82 @@
 ---
 title: Getting Started
-description: Trying ClusterLink for yourself
 weight: 20
 ---
 
+This guide will give you a quick start on installing and setting up the ClusterLink project on a Kubernetes cluster.
+
 ## Prerequisites
 
-<!-- Are there any system requirements for using your project? What languages are supported (if any)? Do users need to already have any software or tools installed? -->
+Before you start, you must have access to a Kubernetes cluster. For example, you can set up a local environment using the [Kind](https://kind.sigs.k8s.io/) project.
```

Comment:
Do you want to provide short instructions for creating cluster(s)?
The user should be able to cut and paste commands from the guide into a shell.

Reply:
I didn't see that anyone else did that, it is a basic assumption that the k8s cluster is running